server/src/models/GameState.ts

```diff
@@ -86,6 +89,12 @@ export class GameState {
         }
     }
 
+    private createFood(team: Team, spawnPoint: MapSpawnPoint, spawnedAt: number) {
+        console.log(`spawning food for ${team}`)
+        const foodID = team
```

Comment:
can there be only one food at a time for a team since it is used as an ID? for minions I used a new uuid()